app/bundles/LeadBundle/Helper/AnonymizeHelper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Helper;
+
+class AnonymizeHelper
+{
+    public const PRE_DEFINED_DOMAIN       = 'ano.nym';
+    public const PRE_PSEUDONYMIZED_DOMAIN = 'pseudo.nym';
+
+    /**
+     * @param string|bool|null $email
+     */
+    public static function email($email, bool $pseudonymized = false, string $newDomain = self::PRE_DEFINED_DOMAIN): string|false
```

Comment:
Can we rename the method name to something like `anonymizeEmail` for more verbosity?